docs/Scaling/overview.md

```diff
@@ -0,0 +1,54 @@
+--8<-- "References/abbreviations.md"
+
+# Scaling Overview
+Scaling functionality in MPS enables Open AMT Cloud Toolkit to support a greater number of managed devices. The toolkit offers various methods for deploying scaling, including Kubernetes (K8s), Azure Kubernetes Service* (AKS), and Docker Swarm*. In addition, administrators can use kuberctl to manage the AKS. 
+
+
+![../assets/images/HighLevelArchitectureScaling](../assets/images/HighLevelArchitectureScaling.png)
+
+**Figure 1: High-level Architecture of Scaling Implementation**
+
+Figure 1 illustrates the basic high-level software flow:
+
+* Managed devices use CIRA to connect and call home to instances of the MPS in the cloud. 
+* RPCs connect to an available instance of the MPS Server with WSS calls. These calls are funneled through Kong* API Gateway, which supports a variety of APIs. Kong manages load balancing, logging, authentication and more. 
```

Comment:
RPCs connect to an available instance of the RPS Server with a secure websocket (WSS) connection. RPS configures managed devices using this connection.